### PR TITLE
[INSTALL.md] Converting lists (of dependencies) to install commands; adding dep install commands for additional distros; updating install/usage information

### PR DESCRIPTION
**Pull request by fusion809**
Hi,

A few improvements for my fellow Linux users to benefit from:

1. Turning Debian and Gentoo dependency lists into a simple one-line command for users to execute. Not sure about other Linux users but I'd much rather a one-line command than a list of deps. I use quotes sometimes when not absolutely necessary if executing these commands in Bash in case users are using another shell like Zsh where quotes may be needed.

2. Adding Arch Linux, Fedora, Mageia and Red Hat Enterprise Linux (RHEL; and rebuilds of it, like CentOS) dependency-installing commands. I know these additional commands work because I build OpenRA packages for these distros in the openSUSE Build Service and these are the build dependencies that my builds need in order to build properly. I also added the runtime dependencies my packages require to this command as I have tried running my packages on these distros and by this I have determined these are the dependencies these packages require in order for OpenRA to run properly.

3. Dropping cURL dependency as now wget also suffices for building OpenRA from the git repo and each of these distros either has cURL or GNU wget pre-installed, quite often as they need it in order for their package manager to fetch packages successfully.

4. Updating the info on running OpenRA after compilation, after all now there are three separate commands for running OpenRA's various included mods.

5. Adding a `sudo` that was missing before the `make install-linux-shortcuts` command.

6. Adding an asterisk after 5.1.5 in the emerge command as the only packages available to satisfy this dependency have a revision number after them (e.g. at the time of writing only `dev-lang/lua-5.1.5-r4`, `-r100` and `-r101` are available from the Portage tree).

I know my commit message isn't perfect, but it's the best I can come up with given how many changes I'm making to this file. If you's would rather I split these changes into a few different commits, so I can use more complete commit messages for each individual change made, I will be happy to oblige. 

Thanks for your time,
Brenton

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -27,43 +27,62 @@ Use `make dependencies` to map the native libraries to your system and fetch the
 
 To compile OpenRA, run `make all` from the command line.
 
-Run with either `launch-game.sh` or `mono --debug OpenRA.Game.exe`.
+Run with either `launch-game.sh` or `mono --debug OpenRA.Game.exe`. It is also possible to specify the mod you wish to run from the command line, e.g. with `./launch-game.sh Game.Mod=ts` if you wish to try the experimental Tiberian Sun mod.
```
Comment (pchote) [inline INSTALL.md]: Can we drop the OpenRA.Game.exe option here? There is no reason for normal players to not be using the launcher script.

Comment (fusion809) [reply INSTALL.md]: Sure, only left it in there because it was there already.

Comment (fusion809) [reply INSTALL.md]: Should I also remove it for macOS while I'm at it?

Comment (pchote) [reply INSTALL.md]: Yeah, sure

**Comment by fusion809** [comment]
Is it also advisable that I keep to 80 characters per line in my extended description or should I drop it all together and just leave this pull request as documentation on why I made the changes?

**Review by Mailaender — DISMISSED** [review]
Looks good to me.

**Comment by fusion809** [comment]
I realized Debian/Ubuntu's APT do not depend on either wget or curl as dependencies and running `apt remove wget curl` on them doesn't cause apt or any other core package to appear in the list of packages to be removed so I'm not certain that they come with curl or wget pre-installed, although I think wget does come pre-installed. The upshot of it is that I added wget to the dependency install command just in case.

**Review by pchote — APPROVED** [review]

**Pull request merged**